css/disciplines-slider.styl

```diff
@@ -43,81 +49,133 @@ DISCIPLINE_COLOR_SOCIAL_SCIENCE = #7EA8CE
+    &:active,
     &.filmstrip--disciplines__discipline-card--active
       background-color: DISCIPLINE_COLOR_CLIMATE
+    &:hover,
+    &:focus
+      color: DISCIPLINE_COLOR_CLIMATE
+      border-color: DISCIPLINE_COLOR_CLIMATE
 
   .filmstrip--disciplines__discipline-card--discipline-history
-    &:hover,
+    &:active,
     &.filmstrip--disciplines__discipline-card--active
       background-color: DISCIPLINE_COLOR_HISTORY
+    &:hover,
+    &:focus
+      color: DISCIPLINE_COLOR_HISTORY
+      border-color: DISCIPLINE_COLOR_HISTORY
     .filmstrip--disciplines__discipline-card__icon
       margin-left: 0.1em
 
   .filmstrip--disciplines__discipline-card--discipline-humanitarian
-    &:hover,
+    &:active,
     &.filmstrip--disciplines__discipline-card--active
       background-color: DISCIPLINE_COLOR_HUMANITARIAN
+    &:hover,
+    &:focus
+      color: DISCIPLINE_COLOR_HUMANITARIAN
+      border-color: DISCIPLINE_COLOR_HUMANITARIAN
 
   .filmstrip--disciplines__discipline-card--discipline-language
     min-width: 4em
-    &:hover,
+    &:active,
     &.filmstrip--disciplines__discipline-card--active
       background-color: DISCIPLINE_COLOR_LANGUAGE
+    &:hover,
+    &:focus
+      color: DISCIPLINE_COLOR_LANGUAGE
+      border-color: DISCIPLINE_COLOR_LANGUAGE
     .filmstrip--disciplines__discipline-card__icon
       margin-left: -1em
 
   .filmstrip--disciplines__discipline-card--discipline-literature
-    &:hover,
+    &:active,
     &.filmstrip--disciplines__discipline-card--active
       background-color: DISCIPLINE_COLOR_LITERATURE
+    &:hover,
+    &:focus
+      color: DISCIPLINE_COLOR_LITERATURE
+      border-color: DISCIPLINE_COLOR_LITERATURE
 
   .filmstrip--disciplines__discipline-card--discipline-medicine
-    &:hover,
+    &:active,
     &.filmstrip--disciplines__discipline-card--active
       background-color: DISCIPLINE_COLOR_MEDICINE
+    &:hover,
+    &:focus
+      color: DISCIPLINE_COLOR_MEDICINE
+      border-color: DISCIPLINE_COLOR_MEDICINE
 
   .filmstrip--disciplines__discipline-card--discipline-nature
-    &:hover,
+    &:active,
     &.filmstrip--disciplines__discipline-card--active
       background-color: DISCIPLINE_COLOR_NATURE
+    &:hover,
+    &:focus
+      color: DISCIPLINE_COLOR_NATURE
+      border-color: DISCIPLINE_COLOR_NATURE
     .filmstrip--disciplines__discipline-card__icon
       margin-left: -0.4em
 
   .filmstrip--disciplines__discipline-card--discipline-physics
-    &:hover,
+    &:active,
     &.filmstrip--disciplines__discipline-card--active
       background-color: DISCIPLINE_COLOR_PHYSICS
+    &:hover,
+    &:focus
+      color: DISCIPLINE_COLOR_PHYSICS
+      border-color: DISCIPLINE_COLOR_PHYSICS
 
   .filmstrip--disciplines__discipline-card--discipline-social-science
-    &:hover,
+    &:active,
     &.filmstrip--disciplines__discipline-card--active
-      background-color: DISCIPLINE_COLOR_SOCIAL_SCIENCE
+      background-color: DISCIPLINE_COLOR_SOCIAL_SCIENCE !important
```

Comment:
Ah good spot, that was something I tried while debugging the social science issue that I left in by mistake. will remove.